src/client/components/feedback/feedback.js

```
@@ -27,115 +27,161 @@ const FEEDBACK_CONFIG = {
  * Feedback form manager class
  */
 class FeedbackManager {
+  /** @type {boolean} */
+  #isOpen = false;
+
+  /** @type {boolean} */
+  #isSubmitting = false;
+
+  /** @type {HTMLElement | null} */
+  #container = null;
```

ESLint

[error] '#container' is defined but never used.

(no-unused-private-class-members)